src/utils/time.py

```diff
@@ -1,8 +1,11 @@
 from datetime import datetime, timezone, timedelta
+import pytz
```

Comment:
Remove the duplicated imports and remove `timezone` and `timedelta` as they aren't used.